modules/search/instant-search/lib/filters.js

```diff
@@ -63,6 +70,32 @@ export function mapFilterToFilterKey( filter ) {
 	return null;
 }
 
+export function mapFilterKeyToFilter( filterKey ) {
+	// TODO: Write a test for this function
```

Comment:
Can we add a test in this PR? 😇

Reply:
👍 I'll add the test after landing and rebasing off of #18119.

Keeping this comment open until then.

Reply:
Tests have now been added; this is ready for another review.